Website/DesktopModules/Hotcakes/Hotcakes.dnn

```diff
@@ -215,6 +215,11 @@
 							<name>03.09.00.SqlDataProvider</name>
 							<version>03.09.00</version>
 						</script>
+            <script type="Install">
+							<path>Providers\DataProviders\SqlDataProvider</path>
+							<name>03.09.01.SqlDataProvider</name>
+							<version>03.09.01</version>

```

Comment:
@ArielBlanco1990 This update is going to be for the 3.9.0 release, not 3.9.1.

Reply:
@WillStrohl This is now resolved.